tests/integration/routes/test_session.py

```diff
@@ -1,19 +1,35 @@
 import time
 from datetime import datetime, timedelta, timezone
 
+import responses
 from freezegun import freeze_time
 from mock.mock import patch
+from sdc.crypto.key_store import KeyStore
 
 from app.questionnaire.questionnaire_schema import DEFAULT_LANGUAGE_CODE
 from app.services.supplementary_data import SupplementaryDataRequestFailed
 from app.settings import ACCOUNT_SERVICE_BASE_URL, ACCOUNT_SERVICE_BASE_URL_SOCIAL
 from app.utilities.json import json_loads
+from tests.app.services.test_request_supplementary_data import TEST_SDS_URL
 from tests.integration.integration_test_case import IntegrationTestCase
 
 TIME_TO_FREEZE = datetime(2020, 1, 1, 12, 0, 0, tzinfo=timezone.utc)
 EQ_SESSION_TIMEOUT_SECONDS = 45 * 60
 BUSINESS_URL = ACCOUNT_SERVICE_BASE_URL
 SOCIAL_URL = ACCOUNT_SERVICE_BASE_URL_SOCIAL
+TEST_SDS_URL = "http://localhost:5003/v1/unit_data"
+
+mock_supplementary_data_payload_missing_data = {
```

Comment:
same as above I think these would be suitable candidates for a conftest function

Reply:
Done 👍